packages/mcp/src/api/server.ts

```
@@ -0,0 +1,175 @@
+        error: 'Not Found',
+        message: `Endpoint ${req.method} ${req.path} not found`,
+        availableEndpoints: [
+          'GET /',
+          'GET /health', 
+          'GET /api/capabilities',
+          'POST /api/trpc/*',
+        ],
+      })
+    })
+
+    this.app.use((error: Error, req: express.Request, res: express.Response, _next: express.NextFunction) => {
```

ESLint

[error] '_next' is defined but never used.

(@typescript-eslint/no-unused-vars)